system-parachains/coretime/coretime-kusama/src/weights/pallet_broker.rs

```diff
@@ -0,0 +1,518 @@
+//! WASM-EXECUTION: `Compiled`, CHAIN: `Some("coretime-kusama-dev")`, DB CACHE: 1024
+
+// Executed Command:
+// ./target/production/polkadot-parachain
+// benchmark
+// pallet
+// --chain=coretime-kusama-dev
+// --wasm-execution=compiled
+// --pallet=pallet_broker
+// --no-storage-info
+// --no-median-slopes
+// --no-min-squares
+// --extrinsic=*
+// --steps=50
+// --repeat=20
+// --json
+// --header=./cumulus/file_header.txt
+// --output=./cumulus/parachains/runtimes/coretime/coretime-kusama/src/weights/
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+
+use frame_support::{traits::Get, weights::Weight};
+use core::marker::PhantomData;
+
+/// Weight functions for `pallet_broker`.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> pallet_broker::WeightInfo for WeightInfo<T> {
+	/// Storage: `Broker::Configuration` (r:0 w:1)
+	/// Proof: `Broker::Configuration` (`max_values`: Some(1), `max_size`: Some(31), added: 526, mode: `MaxEncodedLen`)
+	fn configure() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 2_462_000 picoseconds.
+		Weight::from_parts(2_552_000, 0)
+			.saturating_add(Weight::from_parts(0, 0))
+			.saturating_add(T::DbWeight::get().writes(1))
+	}
+	/// Storage: `Broker::Reservations` (r:1 w:1)
+	/// Proof: `Broker::Reservations` (`max_values`: Some(1), `max_size`: Some(12021), added: 12516, mode: `MaxEncodedLen`)
+	fn reserve() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `10888`
+		//  Estimated: `13506`
+		// Minimum execution time: 25_494_000 picoseconds.
+		Weight::from_parts(26_063_000, 0)
+			.saturating_add(Weight::from_parts(0, 13506))
+			.saturating_add(T::DbWeight::get().reads(1))
+			.saturating_add(T::DbWeight::get().writes(1))
+	}
+	/// Storage: `Broker::Reservations` (r:1 w:1)
+	/// Proof: `Broker::Reservations` (`max_values`: Some(1), `max_size`: Some(12021), added: 12516, mode: `MaxEncodedLen`)
+	fn unreserve() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `12090`
+		//  Estimated: `13506`
+		// Minimum execution time: 22_299_000 picoseconds.
+		Weight::from_parts(22_911_000, 0)
+			.saturating_add(Weight::from_parts(0, 13506))
+			.saturating_add(T::DbWeight::get().reads(1))
+			.saturating_add(T::DbWeight::get().writes(1))
+	}
+	/// Storage: `Broker::Leases` (r:1 w:1)
+	/// Proof: `Broker::Leases` (`max_values`: Some(1), `max_size`: Some(401), added: 896, mode: `MaxEncodedLen`)
+	/// Storage: `ParachainSystem::ValidationData` (r:1 w:0)
+	/// Proof: `ParachainSystem::ValidationData` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainSystem::LastRelayChainBlockNumber` (r:1 w:0)
+	/// Proof: `ParachainSystem::LastRelayChainBlockNumber` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	fn set_lease() -> Weight {
```

Comment:
Weights look good, Relay chain configuration:
```
		require_weight_at_most: Weight::from_parts(170_000_000, 20_000),
```
worst case call is reserve with 13k ... still fits.

Reply:
These are copied over from Rococo, so there may be some changes when I run bench bot.

I've been putting together some emulated tests for the relay calls in the coretime chain in case things change drastically any time the weights are regenerated, might be worth having a weight sanity check for these too.